index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
```

Comment:
Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contrast check
![screencapture-aditus-io-button-contrast-checker-rawcdn-githack-com-2022-04-05-at-14-14-35-168-2022-04-05-16_15_33](https://user-images.githubusercontent.com/24703171/161774581-29a6261d-5995-4c6b-bac9-e9b872798b67.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)